site/gatsby-site/src/utils/cloudinary.js

```diff
@@ -1,11 +1,22 @@
+// import React, { useState, useRef, useEffect } from 'react';
+// import { AdvancedImage, lazyload } from '@cloudinary/react';
+// import { CloudinaryImage } from '@cloudinary/base';
+// import { format, quality } from '@cloudinary/base/actions/delivery';
+// import { auto } from '@cloudinary/base/qualifiers/format';
+// import { auto as qAuto } from '@cloudinary/base/qualifiers/quality';
+// import config from '../../config';
+// import PlaceholderImage from 'components/PlaceholderImage';
+
```

Comment:
Let's remove the comments here too.

Reply:
Comments have been removed in the last commit.